app/src/main/java/com/openclassrooms/realestatemanager/di/Injection.kt

```diff
@@ -0,0 +1,11 @@
+package com.openclassrooms.realestatemanager.di
+
+import com.openclassrooms.realestatemanager.api.FakeEstateApi
+import com.openclassrooms.realestatemanager.repository.EstateRepository
+
+
+object Injection {
+
+    val estateRepository: EstateRepository = EstateRepository(FakeEstateApi)
```

Comment:
```suggestion
    val estateRepository by lazy { EstateRepository(FakeEstateApi) }
```

Serait très pertinent ici. Ca me permet de t'introduire la notion de "lazy initialization". Je te laisse te documenter dessus et on en parle en session mentorat si besoin ? :)

Reply:
Ok :)